README.md

```diff
@@ -92,9 +129,11 @@ Is injective. In a diagram, the arrow is often written with a hook (as in $\hook
 \subsubsection*{Möbius band}
 \subsubsection*{Mapping cylinder}
 
-\subsubsection*{Neighborhood}
+\subsubsection*{Neighborhood} A Neighborhood of a point $x$, is just an open set
+containing that point $x$ under the given Topology.
```

Comment:
Why capitalize topology here (and above in one other place)?

Reply:
I like how it looks, but I usually miss capitalizing them all.